avocado/utils/software_manager.py

```diff
@@ -82,7 +71,7 @@ def get_package_management(self):
         to find the best supported system.
         """
         list_supported = []
-        for high_level_pm in SUPPORTED_PACKAGE_MANAGERS:
+        for high_level_pm in [*SoftwareManager._get_backends()]:
```

Comment:
Not sure if I'm missing something, but why not this?

```python3
for high_level_pm in SoftwareManager._get_backends():
```

Reply:
@beraldoleal True, it will iterate over the keys anyway. I will change on v2. Thanks!